src/.vuepress/navbar.ts

```diff
@@ -2,8 +2,8 @@ import { navbar } from "vuepress-theme-hope";
 
 export default navbar([
   "/",
-  { text: "User Guide", icon: "user", link: "courses/tir_user_guide/" },
-  { text: "Admin Guide", icon: "admin", link: "courses/tir_admin_guide/" },
-  { text: "Deployment Guide", icon: "keyboard", link: "courses/tir_deployment_guide/" },
-  { text: "Training Videos", icon: "video", link: "courses/tir_training_videos/" }
+  { text: "User Guide", icon: "iconfont icon-profile", link: "courses/tir_user_guide/" },
+  { text: "Admin Guide", icon: "iconfont icon-warn", link: "courses/tir_admin_guide/" },
```

Comment:
why 'icon-warn'?  i don't have a good suggestion atm for what a replacement icon might be.